### PR TITLE
Romanian Translation

### PR DESCRIPTION
**Pull request by NoireHime**
Pretty much the title. I am still new to github and programming in general, so please tell me if I made any mistakes. Also, if you're Romanian and find a typo/better way to translate the information feel free to edit the translation.

### CODE REVIEW

File: src/translations/ro.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "error-occured": "O eroare a împiedicat extensia din a functiona corect: ",
+  "version-update-message": "Folosești ezpp! v{0}",
```
Comment (oamaok) [inline src/translations/ro.json]: just to make sure, this is the message displayed when the extension is updated (this has caused some confusion previously). does this message work when displayed as, for example, "Folosești ezpp! v1.6.0"?

**Comment by oamaok** [comment]
thank you for your translation efforts! I'll probably wait a bit before releasing this as I don't want to spam updates too much, and hopefully gain some more translations in the meanwhile.

**Comment by oamaok** [comment]
also, could you squash your commits together as a single one which says "Add Romanian translation" or similar? I'd like to keep the commit history clean. ([how to squash commits](https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git))

File: src/translations/ro.json (hunk 1 above)
Comment (NoireHime) [reply src/translations/ro.json]: Yes. That's how it should work.

**Comment by NoireHime** [comment]
I'm sorry, but currently I am not at home and won't be until August 24th, so I cannot do that right now. I read into it, and I read that you can select the option to squash when you merge my branch with the official one. Again, I am sorry if I am wrong and I am new to GitHub, so I don't really know what I'm doing.

**Pull request merged**